src/couch_replicator_connection.erl

```diff
@@ -120,9 +124,8 @@ handle_cast({connection_close_interval, V}, State) ->
 % owner crashed
 handle_info({'DOWN', _Ref, process, Pid, _Reason}, State) ->
```

Comment:
All workers will be linked to the owner and will be killed. I don't think it really matters if we kill by pid or ref here.